dumpling/export/config.go

```diff
@@ -555,6 +558,9 @@ const (
 	defaultDumpGCSafePointTTL = 5 * 60
 	defaultEtcdDialTimeOut    = 3 * time.Second
 
+	DefaultCollationCompatible = "loose"
+	StrickCollationCompatible  = "strick"
```

Comment:
Do yolu mean?
```suggestion
	StrictCollationCompatible  = "strict"
```

Reply:
The name in spec is strick, i will confirm with PM, i think it should be strict too.